pyproject.toml

```diff
@@ -42,6 +42,7 @@ Documentation = "https://ml.energy/zeus"
 
 [project.optional-dependencies]
 # One day FastAPI will drop support for Pydantic V1. Then fastapi has to be pinned as well.
+apple = ["zeus-apple-silicon"]
```

Comment:
Could you move this line somewhere below? The comment above this line (about FastAPI and Pydantic V1) should be right above the `pfo` optional dependency definition.